modules/mailer/mail.go

```diff
@@ -52,7 +51,7 @@ func SendUserMail(c *macaron.Context, u *models.User, tpl base.TplName, code, su
 		return
 	}
 
-	msg := NewMessage([]string{u.Email}, subject, body)
+	msg := NewMessage([]string{u.Email}, subject, body, "")
```

Comment:
I'm not always passing an empty string. Check [this](https://github.com/gogits/gogs/pull/3138/files#diff-ca7ee5dc0cfb6781d976e6928f5a9c3eR128).